front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -62,18 +68,16 @@ type ApiErrorType =
       [fieldName: string]: CLError[];
     };
 
-const AdminProjectEventEdit = () => {
-  const { eventId, projectId } = useParams() as {
-    eventId: string;
-    projectId: string;
-  };
+const AdminProjectEventEdit = ({ params }: Props) => {
   const { formatMessage } = useIntl();
   const { mutate: addEvent } = useAddEvent();
-  const { data: event, isInitialLoading } = useEvent(eventId);
+  const { data: event, isInitialLoading } = useEvent(params.id);
```

Comment:
Why switch to the old system here - I think the issue will be fixed if we rename the `eventId` param to `id`, no? The naming mismatch is what is causing it to fail, as far as I can tell.

Reply:
Ah yes you're right. I've changed it now :) :+1: